core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1717,12 +1719,12 @@ object SnappySession extends Logging {
       val nocaching = session.getContextObject[Boolean](
         CachedPlanHelperExec.NOCACHING_KEY).getOrElse(false)
       if (nocaching) {
-        key.invalidatePlan()
+        key.invalidatePlan
```

Comment:
Revert. Method calls returning Unit should have parens as per scalaStyle guidelines (not enforced though).